configs/foreman/2.0.yaml

```diff
@@ -408,32 +409,33 @@
         - rhpkg-simple
         - rpm-build
         - shadow-utils
-  - name: foreman-client-2.0-fedora28
-    based_off: foreman-client-nightly-fedora28
+  - name: foreman-client-2.0-el8
+    based_off: foreman-client-nightly-el8
     helper_tags:
-      foreman-client-2.0-fedora28-override: null
-      foreman-client-2.0-fedora28: null
-    build_target: foreman-client-2.0-fedora28-build
+      foreman-client-2.0-el8-override: null
+      foreman-client-2.0-el8: null
+    build_target: foreman-client-2.0-el8-build
     arches:
       - x86_64
     inherits:
-      foreman-client-2.0-fedora28-build:
-        foreman-client-2.0-fedora28-override: 0
-      foreman-client-2.0-fedora28-override:
-        foreman-client-2.0-fedora28: 0
-      foreman-client-2.0-fedora28: {}
+      foreman-client-2.0-el8-build:
+        foreman-client-2.0-el8-override: 0
+      foreman-client-2.0-el8-override:
+        foreman-client-2.0-el8: 0
+      foreman-client-2.0-el8: {}
     external_repos:
-      - fedora-28-updates
-      - fedora-28
+      - centos8-baseos
+      - centos8-appstream
     build_groups:
       build:
         - bash
         - bzip2
+        - centos-release
         - coreutils
         - cpio
         - diffutils
-        - fedora-release
         - findutils
+        - foreman-build
```

Comment:
Do we include this in client repos?

Reply:
I copied what we have in nightly. My best guess is no, but what do I know ;)

Reply:
Would you like to block this until we find out, or postpone that?

Reply:
Not a blocker I think

Reply:
added a card to our backlog